### PR TITLE
VSFP-13: Research Eros

### PR DESCRIPTION
**Pull request by fl3pp**

### CODE REVIEW

File: SA/content/tool_research/eros.typ [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+implements this technique. Eros is particuarly suited to creative people with an
+artistic interest.
+
+#figure(
+  image("../../static/eros_screenshot.png", width: 60%),
+  caption: "Screenshot of Eros")
+
+"Tangible Functional Programming" is a very interesting technique, and
+particularly the way Eros visualized pure values can be an inspiration for
+VisualFP. Ultimately, we feel like the technique is not well suited for general
```
Comment (lstreckeisen) [inline SA/content/tool_research/eros.typ:26]:
```suggestion
VisualFP. Ultimately the technique is not well suited for general

```

File: SA/content/tool_research/eros.typ [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+purpose functional programming and education.
```
Comment (lstreckeisen) [inline SA/content/tool_research/eros.typ:27]:
```suggestion
purpose functional programming, especially in an educational context. 

```

File: SA/content/tool_research/eros.typ [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+apply values to each another even if they are nested in functions or tuples.
+This is called _deep application_.
+
+Along with the technique, Conal developed an application called Eros that
+implements this technique. Eros is particuarly suited to creative people with an
+artistic interest.
+
+#figure(
+  image("../../static/eros_screenshot.png", width: 60%),
+  caption: "Screenshot of Eros")
```
Comment (lstreckeisen) [inline SA/content/tool_research/eros.typ:22]: Declare source of image

Comment (fl3pp) [reply SA/content/tool_research/eros.typ]: added source

**Pull request merged**